Assignment02/main.js

```diff
@@ -0,0 +1,195 @@
+        lastName.value = formatString; //setting form input value to formatted string
+    });
+
+    age.addEventListener("blur", () => { //on blur event of age input
+        let ageEntered = parseInt(age.value); //converting string value to integer
+        if(Number.isInteger(ageEntered) == false){ //checking if entry value is not a number
+            alert("Please input only a number for age!"); //alerting user that they must enter an integer
+            age.value = 0; //setting age value in the form to 0 -- this is the only way I could stop an endless loop of alerts for the event
+            age.focus(); //setting cursor focus to age input field
+        }
+    });
+
+    rating.addEventListener("change", () => { //event listener added to check value of meal rating upon a change
+        switch(rating.value){ //switch statement based on value of meal rating selection
+            case "": //case that there was no entry
+                if(commentBox.getAttribute("class") === "visible"){ //checking if commentBox is visible
+                    commentBox.classList.remove("visible"); //removing visible class
+                    commentBox.classList.add("hidden"); //adding hidden class
+                }
+                commentBox.innerHTML = ""; //resetting commentBox to blank
+                break; // breaking out of switch
+            case "0": //if user selects zero stars
+                commentBox.innerHTML = ""; //resetting commentBox inner HTML to blank
+                let commentLabel = document.createElement("label"); //creating label element
+                commentLabel.innerHTML = "You rated us zero stars! Please let us know why:"; //setting text inside label element
+                let commentBreak = document.createElement("br");//creating line break element
+                let commentText = document.createElement("textarea"); //creating textarea  element
+                commentText.setAttribute("id", "userComments"); //setting textarea id 
+                commentText.setAttribute("name", "userComments"); //setting textarea name 
+                commentText.setAttribute("rows", "5"); //setting textarea rows
+                commentText.setAttribute("cols", "33"); //setting textarea columns
+                commentBox.appendChild(commentLabel); //appending label element to commentBox div
+                commentBox.appendChild(commentBreak); //appending linebreak element to commentBox div
+                commentBox.appendChild(commentText); //appending textarea element to commentBox div
+                commentBox.setAttribute("class", "visible"); //setting div to visible
+                break; //break out of switch statement
+            case "1": //case 1
+            case "2": //case 2 ***** all cases 1-4 will be executing case 4 code
+            case "3": //case 3
+            case "4": //case 4
```

Comment:
since you have multiple cases that all execute the same - this works. Would an if be easier to read? I could see a case either for keeping the switch or moving to an if.